config/crd/bases/camel.apache.org_builds.yaml

```diff
@@ -19,7 +19,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.1
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
This does not look good. I think you have some controller-gen version in your local environment different than the one we are using.

Reply:
I did wonder.. fixed.